static_src/components/count_status.jsx

```diff
@@ -37,7 +38,12 @@ export default class CountStatus extends React.Component {
 
     return (
       <span className={ this.styler('count_status', statusClass) }>
-        <strong>{ props.count }</strong> { props.name }
+        <span style={{ float: 'left' }}>
```

Comment:
I'd like to avoid the one-off styles and keep everything in `cg-style` and use class names. This keeps everything re-usable and documented in the pattern library. Also, forcing ourselves to name these kinds of things helps us keep our design patterns clean and consistent.